packages/genesys-spark-components/src/components/stable/gux-toggle/gux-toggle-slider/gux-toggle-slider.scss

```diff
@@ -137,6 +138,12 @@ gux-toggle-slider {
           }
         }
       }
+
+      &.gux-rtl {
+        .gux-switch {
+          transform: translateX($slider-transition-rtl);
+        }
+      }
```

Comment:
manual change

Reply:
This is what it looks like without it. Prior to the changes to logical properties it looked fine just not mirrored. 

https://github.com/user-attachments/assets/5bb179be-42e9-4b10-950c-0a8d69cfeac7